src/Behat/Behat/Context/Reader/AnnotatedContextReader.php

```diff
@@ -196,7 +196,7 @@ private function isEmpty($docLine)
      */
     private function isNotAnnotation($docLine)
     {
-        return '@' !== substr($docLine, 0, 1);
+        return '@' !== $docLine[0];
```

Comment:
PHPStorm suggests this as an improvement? This makes it less clear to me, I know this works and that you can retrieve individual characters in a string by accessing them using the array key syntax, but the original code is much easier to understand for a human being.

Reply:
and this code breaks for an empty string

Reply:
Covered here:
```php
           if ($this->isEmpty($docLine)) {
                continue;
            }
            if ($this->isNotAnnotation($docLine)) {
                continue;
            }
```
I'd suggest even to simplify `if ($this->isEmpty($docLine) || $this->isNotAnnotation($docLine)) ...`